File: JavaServer/Player/Player.java
package JavaServer.Player;

import JavaServer.TokensPrototype.Token;

public class Player {
    private int ID;
    private int lives = 3;
    //Agregar atributo Reto actual
    private Token currentToken;
    private int score;
    private int time; 
    public Player(int iD, int lives, Token currentToken, int score, int time) {
            ID = iD;
            this.lives = lives;
            this.currentToken = currentToken;
            this.score = score;
            this.time = time;
        }
        
    public int getID() {
        return ID;
    }

    public void setID(int iD) {
        ID = iD;
    }

    public int getLives() {
        return lives;
    }

    public void setLives(int lives) {
        this.lives = lives;
    }

    public Token getCurrentToken() {
        return currentToken;
    }

    public void setCurrentToken(Token currentToken) {
        this.currentToken = currentToken;
    }

    public int getScore() {
        return score;
    }

    public void setScore(int score) {
        this.score = score;
    }

    public int getTime() {
        return time;
    }

    public void setTime(int time) {
        this.time = time;
    }

    
    

}

File: src/main/java/cr/ac/tec/PythonClient/GameFeatures/Player.py
class Player():
    def __init__(self, ID, currentToken = null, score = 0, treeList = []):
        super().__init__()
        self.ID = ID
        self.lives = 3
        self.currentToken = currentToken
        self.score = score
        self.treeList = treeList
        self.currentPowerUp = currentPowerUp
    
    def getID(self):
        return self.ID

    def getLives(self):
        return self.lives
    
    def getCurrentToken(self):
        return self.currentToken

    def getScore(self):
        return self.score
    
    def getTreeList(self):
        return self.treeList
    
    def getCurrentPowerUp (self):
        return self.currentPowerUp

    def setLives(self, lives):
        self.lives = lives

    def setCurrentToken(self, currentToken):
        self.currentToken = currentToken

    def setScore(self, score):
        self.score = score

    def setTreeList(self, treeList):
        self.treeList = treeList 

    def setCurrentPowerUp (self, currentPowerUp):
        self.currentPowerUp = currentPowerUp   